src/store/modules/user/actions.ts

```diff
@@ -125,6 +154,17 @@ const actions: ActionTree<UserState, RootState> = {
     */
      setUserInstanceUrl ({ state, commit }, payload){
       commit(types.USER_INSTANCE_URL_UPDATED, payload)
-    }
+    },
+
+  /**
+   * Set Ecom store
+   */
+    async setEcomStore({ commit, dispatch }, payload) {
```

Comment:
I think already have an action `setSelectedBrand` for the same. Same for getters. I think we could use the EcomStore for consistency across the apps, we could use Brand label on settings page, but use eCom store across the app

Reply:
Got it sir. Fixed it